addons/zeus/config.cpp

```diff
@@ -25,6 +25,7 @@ class CfgPatches {
             QGVAR(moduleTeleportPlayers),
             QGVAR(moduleHeal),
             QGVAR(moduleSuicideBomber),
+            QGVAR(moduleBurn),
```

Comment:
Should be removed since it is in the units array of the `GVAR(fire)` class.

Reply:
I was wondering about this, because other modules are listed in 2 different `units[]`

```suggestion
```